daemon/cmd/ciliumendpoints.go

```diff
@@ -46,8 +50,9 @@ func (d *Daemon) cleanStaleCEPs(ctx context.Context, eps localEndpointCache, cil
 			}
 			for _, cep := range ces.Endpoints {
 				if cep.Networking.NodeIP == node.GetCiliumEndpointNodeIP() && eps.LookupCEPName(ces.Namespace+"/"+cep.Name) == nil {
-					d.deleteCiliumEndpoint(ctx, ces.Namespace, cep.Name, nil, ciliumClient, eps,
-						enableCiliumEndpointSlice)
+					if err := d.deleteCiliumEndpoint(ctx, ces.Namespace, cep.Name, nil, ciliumClient, eps, enableCiliumEndpointSlice); err != nil {
+						errs = append(errs, err)
```

Comment:
nice, I like the idea of still trying to attempt cleanup of other CEPs even if one fails.